### PR TITLE
docs(VNavigationDrawer): add api descriptions

### PR DESCRIPTION
**Pull request by webdevnerdstuff**
## Description
Added `railWidth`, `width`, and `location` descriptions.

## Markup:
n/a

### CODE REVIEW

File: packages/api-generator/src/locale/en/v-navigation-drawer.json [hunk 1]
```diff
@@ -17,10 +17,10 @@
     "touchless": "Disable mobile touch functionality",
     "value": "Controls whether the component is visible or hidden.",
     "rail": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-navigation-drawer.json))",
-    "railWidth": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-navigation-drawer.json))",
+    "railWidth": "Designates the width assigned when the `rail` prop is turned on",
```
Comment (KaelWD) [inline packages/api-generator/src/locale/en/v-navigation-drawer.json:20]:
```suggestion
    "railWidth": "Sets the width for the component when `rail` is enabled.",
```

File: packages/api-generator/src/locale/en/v-navigation-drawer.json [hunk 2]
```diff
@@ -17,10 +17,10 @@
...
     "scrim": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-navigation-drawer.json))",
-    "width": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-navigation-drawer.json))",
-    "location": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-navigation-drawer.json))",
+    "width": "Sets the width for the component.",
+    "location": "Apply the position navigation drawer on the screen.",
```
Comment (KaelWD) [inline packages/api-generator/src/locale/en/v-navigation-drawer.json:23]:
```suggestion
    "location": "Sets where the drawer should appear in the layout.",
```

**Review by KaelWD — CHANGES_REQUESTED** [review]

**Comment by webdevnerdstuff** [comment]
Suggestions committed.

File: packages/api-generator/src/locale/en/v-navigation-drawer.json [hunk 3]
```diff
@@ -17,10 +17,10 @@
     "touchless": "Disable mobile touch functionality",
     "value": "Controls whether the component is visible or hidden.",
     "rail": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-navigation-drawer.json))",
-    "railWidth": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-navigation-drawer.json))",
+    "railWidth": "Sets the width for the component when `rail` is enabled.",
     "scrim": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-navigation-drawer.json))",
-    "width": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-navigation-drawer.json))",
-    "location": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-navigation-drawer.json))",
+    "width": "Sets the width for the component.",
```
Comment (johnleider) [inline packages/api-generator/src/locale/en/v-navigation-drawer.json:22]:
```suggestion
```

This already exists within **generic.json**

File: packages/api-generator/src/locale/en/v-navigation-drawer.json [hunk 4]
```diff
@@ -17,10 +17,10 @@
...
+    "location": "Sets where the drawer should appear in the layout.",
```
Comment (johnleider) [inline packages/api-generator/src/locale/en/v-navigation-drawer.json:23]:
```suggestion
    "location": "Sets the drawer's location within the layout.",
```

**Review by johnleider — CHANGES_REQUESTED** [review]

**Review by johnleider — APPROVED** [review]

**Pull request merged**